### PR TITLE
🔀🌐🇫🇷 Add Update "Maintien de l'Ordre"

### PR DESCRIPTION
**Pull request by deoxis9001**
I create if people want to help^^
not finished

### CODE REVIEW

**Comment by deoxis9001** [comment]
sanny can check please ??

File: src/i18n/fr_FR.ts [hunk 1]
```diff
@@ -616,6 +616,84 @@ Si plusieurs demandes échouent dans un court laps de temps, cela peut être dû
             special:
                 'Remorqueuse nécessaire (FPT, FPTL, CCFS, CCFM, CCFL, CCRL, CCRM, CCGC, FMOGP, VLHR, CDHR, FDGP)',
         },
+        39: {
+            caption: 'VR',
+            coins: 10,
+            credits: 10_000,
+            minPersonnel: 0,
```
Comment (Suchty112) [inline src/i18n/fr_FR.ts:623]: MinPersoneel is always 1 person. Not 0

File: src/i18n/fr_FR.ts [hunk 2]
```diff
@@ -616,6 +616,84 @@ Si plusieurs demandes échouent dans un court laps de temps, cela peut être dû
...
+                    },
+                },
+            },
+        },
+        41: {
+            caption: 'RAM',
+            coins: 10,
+            credits: 25_000,
+            minPersonnel: 0,
+            maxPersonnel: 0,
```
Comment (Suchty112) [inline src/i18n/fr_FR.ts:647]: Max is 2
image

File: src/i18n/fr_FR.ts [hunk 3]
```diff
@@ -616,6 +616,84 @@ Si plusieurs demandes échouent dans un court laps de temps, cela peut être dû
             special:
                 'Remorqueuse nécessaire (FPT, FPTL, CCFS, CCFM, CCFL, CCRL, CCRM, CCGC, FMOGP, VLHR, CDHR, FDGP)',
         },
+        39: {
+            caption: 'VR',
+            coins: 10,
```
Comment (Suchty112) [inline src/i18n/fr_FR.ts:621]: coins are 5

File: src/i18n/fr_FR.ts [hunk 4]
```diff
@@ -616,6 +616,84 @@ Si plusieurs demandes échouent dans un court laps de temps, cela peut être dû
...
+                        all: true,
+                    },
+                },
+            },
+		},
+        44: {
+            caption: 'ReBP',
+            coins: 10,
+            credits: 10_000,
+            minPersonnel: 0,
```
Comment (Suchty112) [inline src/i18n/fr_FR.ts:683]: Here it fits with the staff

**Review by Suchty112 — CHANGES_REQUESTED** [review]
Otherwise I have nothing more for now.
But do you also want to do the other modules? or will you just edit this file?

**Comment by deoxis9001** [comment]
Can I let you do the other files?

File: src/i18n/fr_FR.ts (hunk 1 above)
Comment (deoxis9001) [reply src/i18n/fr_FR.ts]: thank you 
I have modified

**Comment by Suchty112** [comment]
> Can I let you do the other files?

Yes, I'll do the other files. I still have to do V3 too ^^

File: src/i18n/fr_FR.ts (hunk 2 above)
Comment (deoxis9001) [reply src/i18n/fr_FR.ts]: thank you 
I have modified

File: src/i18n/fr_FR.ts (hunk 3 above)
Comment (deoxis9001) [reply src/i18n/fr_FR.ts]: thank you 
I have modified

File: src/i18n/fr_FR.ts (hunk 4 above)
Comment (deoxis9001) [reply src/i18n/fr_FR.ts:683]: thank you 
I have modified

**Review by Suchty112 — APPROVED** [review]

**Pull request merged**